<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="inheritxsales" name="myTestCases.Hybrid_ExecuteTestCases_dataprovider" tests="1" failures="0" timestamp="3 Mar 2016 08:46:34 GMT" time="0.000" errors="1">
  <testcase name="HybridExecuteClass" time="0.000" classname="myTestCases.Hybrid_ExecuteTestCases_dataprovider">
    <error type="java.lang.RuntimeException" message="java.io.FileNotFoundException: /home/aarti/workspace/Hybrid driven Testing FrameworkSelenium/src/TestCase.xlsx (No such file or directory)">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: /home/aarti/workspace/Hybrid driven Testing FrameworkSelenium/src/TestCase.xlsx (No such file or directory)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:162)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:112)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:176)
Caused by: java.io.FileNotFoundException: /home/aarti/workspace/Hybrid driven Testing FrameworkSelenium/src/TestCase.xlsx (No such file or directory)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(FileInputStream.java:146)
	at myExcelFileReader.ExcelReaderFile.ReadDataFromExcelFile(ExcelReaderFile.java:22)
	at myTestCases.Hybrid_ExecuteTestCases_dataprovider.dataProviderClass(Hybrid_ExecuteTestCases_dataprovider.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:136)
	... 20 more
]]>
    </error>
  </testcase> <!-- HybridExecuteClass -->
</testsuite> <!-- myTestCases.Hybrid_ExecuteTestCases_dataprovider -->
